test/servico/cst09Test.java

```diff
@@ -53,25 +44,31 @@ public static void tearDownClass() {
     public void setUp() throws ParseException, NonexistentEntityException {
         //Setup Declaração de Ausência Professor 2
         ProfessorService professorService = new ProfessorService();
-        ProfessorModel professor = professorService.obterProfessorPorNome("Professor2");
+        ProfessorModel professor = professorService.obterProfessorPorNome(
+                "Professor2");
         
         NotificacaoService notificaçãoService = new NotificacaoService();
-        notificaçãoService.notificarAusencia(professor.id, "25/11/2013 18:30", "25/11/2013 19:59", "Motivo Declarado", new LinkedList<String>());
+        notificaçãoService.notificarAusencia(
+                professor.id, "25/11/2013 18:30", "25/11/2013 19:59", 
+                "Motivo Declarado", new LinkedList<String>());
         
         //Setup Professor 3 como substituto
         List<AusenciaModel> ausencias = notificaçãoService.listarAusencias();    
-        notificaçãoService.definirSubstituto(ausencias.get(0).codigo,"Professor3");
+        notificaçãoService.definirSubstituto(
+                ausencias.get(0).codigo,"Professor3");
     }
     
     @After
     public void tearDown() {
     }
     
     @Test
-    public void testeEfetuarRealocação_SemSubstitutosDisponíveis() throws ParseException
+    public void testeEfetuarRealocação_SemSubstitutosDisponíveis() 

```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Method names should not contain underscores](https://www.codacy.com/app/calebepb/substituicao/file/2767610636/issues/source?bid=3308597&fileBranchId=3353664#l66)